embark-ui/src/components/TransactionDecoder.js

```diff
@@ -67,7 +74,7 @@ class TransactionDecoder extends React.Component {
 }
 
 TransactionDecoder.propTypes = {
-  history: PropTypes.array,
+  history: PropTypes.object,
```

Comment:
This produces an error at runtime on latest `master`. Will make a separate fix commit for this.